src/components/Home.js

```diff
@@ -0,0 +1,35 @@
+import React, { useEffect, useState } from 'react';
+import { useHistory } from 'react-router-dom';
+
+import getToken from '../lib/tokens';
+
+const Home = () => {
+  const [token, setToken] = useState(
+    JSON.parse(localStorage.getItem('token' || '')),
+  );
+
+  const history = useHistory();
+
+  useEffect(() => {
+    if (token) {
+      history.push('/list-view');
+    }
+    localStorage.setItem('token', JSON.stringify(token));
+  }, [token, history]);
+
+  const handleClick = () => {
+    if (!token) {
```

Comment:
Also, I don't think there is a need of adding an if statement in the handleClick function. I think the useEffect hook is already doing that job before returning anything.

Reply:
Thanks for pointing this out! We'll work on it.

Reply:
Thanks for pointing this out! We'll work on it.